client/src/store.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
+import rootReducer from './reducers';
+import rootSaga from './sagas';
+
+const sagaMiddleware = createSagaMiddleware();
+
+const initialState = {};
+const middleware = [sagaMiddleware];
```

Comment:
should probably just include `sagaMiddleware` directly in `applyMiddleware` since we don't have any others atm